prod/westeurope/linux/appbackendli/app_service/terragrunt.hcl

```diff
@@ -147,7 +152,8 @@ inputs = {
     TOKEN_DURATION_IN_SECONDS = "2592000"
 
     // FUNCTIONS
-    API_URL       = "NONE"
+    // this function shouldn't be called anymore by the appbackendli.
+    API_URL       = "http://${dependency.functions_app1_r3.outputs.default_hostname}/api/v1"
```

Comment:
Can we safe remove API_URL?

Reply:
I had a quick chat with @balanza : it can't be removed. Previously I put the string "NONE" but we decided it's safer to have a valid url .... 
Don't you like it?  BTW I put a comment to highlight the fact that url should be called.

Reply:
thanks @uolter